src/entries/popup/hooks/useNativeAsset.ts

```diff
@@ -1,23 +1,52 @@
-/* eslint-disable no-nested-ternary */
 import { AddressZero } from '@ethersproject/constants';
 import { Address, useNetwork } from 'wagmi';
 
+import { NATIVE_ASSETS_MAP_PER_CHAIN } from '~/core/references';
 import { useUserTestnetNativeAsset } from '~/core/resources/assets/userTestnetNativeAsset';
 import { useCurrentAddressStore, useCurrentCurrencyStore } from '~/core/state';
+import { ParsedUserAsset } from '~/core/types/assets';
 import { ChainId } from '~/core/types/chains';
-import { isCustomChain } from '~/core/utils/chains';
+import { chainNameFromChainId, isCustomChain } from '~/core/utils/chains';
 
 import { useCustomNetworkAsset } from './useCustomNetworkAsset';
 import { getNetworkNativeAssetUniqueId } from './useNativeAssetForNetwork';
+import { useNativeAssets } from './useNativeAssets';
 import { useUserAsset } from './useUserAsset';
 
+const useMockNativeAssets = ({
+  chainId,
+}: {
+  chainId: ChainId;
+}): ParsedUserAsset | undefined | null => {
+  const nativeAssets = useNativeAssets();
+  const { chains } = useNetwork();
+  if (!nativeAssets) return null;
+  return chains
+    .map((chain) => {
+      const assetKey = `${NATIVE_ASSETS_MAP_PER_CHAIN[chain.id]}_${
+        ChainId.mainnet
```

Comment:
is mainnet right?

Reply:
yes this is coming from here https://github.com/rainbow-me/browser-extension/blob/3d5e28eed26cc6e8d6c6ee4ec1b7e1e8bcd362eb/src/entries/popup/hooks/useNativeAssets.ts#L35C21-L35C21